ui.content/src/main/content/jcr_root/conf/acm/settings/snippet/available/core/argument/color.yml

```diff
@@ -0,0 +1,12 @@
+group: Argument
+name: argument_color
+content: |
+  args.color("${1:name}") { label = "${2:label}"; value = ${3:value}; rgba() }
```

Comment:
snippet should be present the most popular usage; skip `rgba()` as it is default